### ISSUE TITLE
Proxy Server Support?

### ISSUE THREAD

**Issue by fuzzygroup** [feedjira/feedjira#347]
Hi, 

I've been using a vendor gem'd version of Feedzirra for, well, forever.  And we're reliant on the (partially working) proxy server support.  I can't find any mention of proxy servers in feedjira -- does support for using feedjira thru a proxy server exist?

Thanks
Scott

**Comment by PascalTurbo** [comment]
I'll have a look. Should be possible to hand over the proxy configuration to faraday.
This depends on #340 - I'm on it.

**Comment by fuzzygroup** [comment]
There's actually a relatively easy work around -- separate out your network fetch from your feed parsing.  I solved this by using Mechanize to fetch the data since Mechanize works very well with proxies and then I fed it into feedjira for parsing.  Works great.

By the way thank you for your good work on this gem.  Very much appreciated.

So while I think proxy support makes sense there is an easy work around (and that's why I made this comment because there should be a mention of the work proxy somewhere in the code base).
